make-syncthing.bash

```diff
@@ -7,15 +7,19 @@ JNIDIR="/src/main/jniLibs"
 
 case "$1" in
     arm)
-        export CGO_ENABLED=0
-        export GOOS=linux
+        export CGO_ENABLED=1
+	# The executable "arm-linux-androideabi-gcc" can be found in the bin folder of your standalone android ndk toolchain
+	export CC=arm-linux-androideabi-gcc
+        export GOOS=android
```

Comment:
Indentation.
Also, what is that toolchain is not available?

We should check the existence of the toolchain and print a friendly error.